src/Map.tsx

```diff
@@ -57,23 +68,38 @@ export const Map = ({
 									iconAnchor: [10, 30],
 									html: renderToString(<MarkerIcon style={{ color }} />),
 								})}
-								position={origin.position}
-							/>
+								position={origin.position}>
+								<Popup offset={[0, -15]}>
+									{name}<br/>
+									Weight: {numberFormatter.format(weight)} kg<br/>
+									Value: {currencyFormatter.format(value)}
+								</Popup>
+							</Marker>
 							<Marker
 								icon={L.divIcon({
 									className: '',
 									iconSize: [30, 30],
 									iconAnchor: [15, 30],
 									html: renderToString(<ParcelIcon style={{ color }} />),
 								})}
-								position={destination.position}
-							/>
+								position={destination.position}>
+								<Popup offset={[0, -15]}>
```

Comment:
Instead of setting the offset on the Popup, use the `popupAnchor` propery on the Icon: https://leafletjs.com/reference-1.6.0.html#icon-popupanchor

This allows to swap out the icon, without needing to update the Popup definition.